manifests/d/Dell/DisplayManager/1.56.2110/Dell.DisplayManager.locale.en-US.yaml

```
@@ -0,0 +1,31 @@
+PackageIdentifier: Dell.DisplayManager
+PackageVersion: 1.56.2110
+PackageLocale: en-US
+Publisher: Dell Inc.
```

```suggestion
Publisher: EnTech Taiwan
```
This should match Control Panel